### ISSUE TITLE
Fatal Include Error

### ISSUE THREAD

**Issue by trevorsb** [h2zero/NimBLE-Arduino#409]
Hi,

I use NimBLE-Arduino Library with ESP32-WROOM-32E. I port it to my chip through PlatformIO. It has worked very well up until today. Today, when I try to compile my code, I get an include error `.pio/libdeps/esp32dev/NimBLE-Arduino/src/nimble/nimble/host/src/ble_hs_hci_cmd.c:31:26: fatal error: soc/soc_caps.h: No such file or directory.
compilation terminated.` 

I've tried to compile several of my branches that were compiling without error just yesterday, and I get this error every time. The only thing I can think of is some sort of versioning issue, where something has been updated or gone out of date. 

Here is my `platformio.ini`:

```
[env:esp32dev]
platform = espressif32
board = esp32dev
framework = arduino
board_build.partitions = large_spiffs_16MB.csv
;board_build.partitions = no_ota.csv
monitor_filters = esp32_exception_decoder
build_type = debug
monitor_speed = 115200
lib_deps =
  https://github.com/schreibfaul1/ESP32-audioI2S/
  https://github.com/mobizt/Firebase-ESP-Client
  https://github.com/Chris--A/Keypad
  https://github.com/Marzogh/SPIMemory
  https://github.com/bblanchon/ArduinoJson
  https://github.com/GoogleCloudPlatform/google-cloud-iot-arduino
  https://github.com/256dpi/arduino-mqtt
  https://github.com/h2zero/NimBLE-Arduino
```

Any ideas would be appreciated. Thank you

![Screen Shot 2022-05-31 at 12 05 02 PM](https://user-images.githubusercontent.com/48928187/171255006-41c700ff-9759-4668-bc7f-c892dc66a715.png)
!

**Comment by h2zero** [comment]
Looks like you either need to delete your pio folder or update to the newer arduino-esp32. Barring that you may want to use the 1.3.8 release instead of the master branch. I'll have to fix this for the older arduio-esp32 versions.

**Comment by trevorsb** [comment]
Hey h2zero, thank you for replying! I tried deleting the `.pio` folder, but n success.

Could you give me a quick explanation of how to update to the newer arduino-esp32 or to pull back to the 1.3.8 release? I'm sorry, I know this is simple, but not my wheelhouse. Thank you!

**Comment by trevorsb** [comment]
Update: I took the momentous step and googled it - updating arduino-esp32 resolved the issue!

Thank you @h2zero !

**Comment by h2zero** [comment]
Lol, great!

**Comment by h2zero** [comment]
Resolved in [726b405](https://github.com/h2zero/NimBLE-Arduino/commit/726b405ac45399d02250543e2e97c263467c967a)
